logName: 0.1
logPlanner: 1